src/core/server/saved_objects/routes/pit.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { IRouter } from '../../http';
+import { CoreUsageDataSetup } from '../../core_usage_data';
+
+interface RouteDependencies {
+  coreUsageData: CoreUsageDataSetup;
+}
+
+const TYPE_DELIMITER = ',';
+
+export const registerPointInTimeRoute = (router: IRouter, { coreUsageData }: RouteDependencies) => {
+  router.post(
+    {
+      path: '/{types}/_pit',
```

Comment:
In this case I was trying to match the way the PIT API works in ES, however upon looking at it again, I realize they are sending the `keep_alive` in the query which I neglected to do here anyway. 🙄 

I agree it would simplify the logic not to have to deal with comma-separated types - will plan to revert this, but let's wait first to see what @rudolf thinks about whether we need this exposed to the client in the first place.